### DISCUSSION TITLE
Use Melody project, it's better than Os include Music

### DISCUSSION THREAD

**Discussion by zachari043** [elementary/triage#250] [Ideas]
https://github.com/artemanufrij/playmymusic

I think it would be better tu make this the official music player for elementary OS !

![image-1_orig](https://github.com/elementary/triage/assets/30619426/7d8064b0-008b-47e2-98d0-905728a7bb78)

**Comment by ghost** [comment]
Music app originally had this kind of iTunes like UI - the problem is it's hard for a team like Elementary to maintain such big apps.

In my opinion, the current Music app is better. It just plays music. No library, no anything, just music. The other music player distracted me a ton.

HOWEVER, it is good to see alternatives that people like!

**Reply by zachari043** [reply]
"it's hard for a team like Elementary to maintain such big apps" contact devloper to integrate this and give up Music so you will get more time for more critical task

**Reply by stsdc** [reply]
Who will  support it after integration? Even initial developer stopped updating the app

**Comment by danirabbit** [comment]
Hey thanks for the suggestion! I’m going to close this discussion since we don’t replace our own first party apps with third party apps. You’re always welcome to report issues or request features for Music on its repo at https://github.com/elementary/music
